resource-directory/service/filterBitmask.go

```diff
@@ -44,13 +44,12 @@ func filterPendingCommandToBitmask(f pb.GetPendingCommandsRequest_Command) filte
 }
 
 func filterPendingsCommandsToBitmask(commandFilter []pb.GetPendingCommandsRequest_Command) filterBitmask {
```

Comment:
maybe you can use filterBitmask from https://github.com/plgd-dev/cloud/blob/v2/grpc-gateway/subscription/filterBitmask.go

Reply:
Yes, it's the same. Switched to use the grpc-gw/subscription code.